### ISSUE TITLE
Tabs don't record value

### ISSUE THREAD

**Issue by adavie1** [daief/daisyui-vue#3]
**Environment:**

- JavaScript
- Vite 3.0.0
- Vue 3.2.37
- Tailwind 3.2.1
- daisyui 2.33.0

This is my code:

```
    <Tabs v-model="currentTab" type="boxed" size="xs">

      <Tab name="rta" title="Acquire">
      Acquire
      </Tab>
      <Tab name="rts" title="Share">
       RTS
      </Tab>

      <Tab name="rtu" title="Upgrade" />
      <Tab name="rtb" title="Backhaul" />
    </Tabs>
...
export default {
  data () {
    return {
      currentTab: 'rts'
    }
  }
}

```
The `currentTab` doesn't change. When follow the sample code:
In script **setup**:
` const currentTab = ref('rts');`

`<Tabs v-model={[currentTab.value, 'value']}>`
I get an error:
![image](https://user-images.githubusercontent.com/3030402/199946719-ef468b6e-f3d1-4fcd-966b-af4d9e756167.png)

The value of `currentTab` doesn't change:
![image](https://user-images.githubusercontent.com/3030402/199947121-589232db-50c5-4b45-a5de-64c0f6a550dd.png)

Select Upgrade:
![image](https://user-images.githubusercontent.com/3030402/199947328-1f20d7f8-efe9-4df2-8316-462f9c075f46.png)

**Comment by daief** [comment]
Thank you for your feedback.

`Tabs` does has some problem. But I am a little busy recently, I will fix it when I have time.

But it is ok in normal case, demo: <https://codesandbox.io/s/daisyui-vue-3-lp7i8d>

Secondly, `<Tabs v-model={[currentTab.value, 'value']}>` is a [jsx](https://github.com/vuejs/babel-plugin-jsx#v-model) syntax that can not be written in template.

**Comment by adavie1** [comment]
No problem. Ah JSX syntax, no wonder I didn't recognise it! Thought TS thing or Vue 3 thing.

**Comment by daief** [comment]
The problem is fixed(0.0.1-alpha.12).

But there are two breaking changes(I am sorry this library is in early development):

- just use `v-model` for `Tabs`, no more `v-model:value`
- rename `Tab` to `TabPanel`
